src/config/logger.ts

```diff
@@ -3,6 +3,7 @@ import bformat from 'bunyan-format';
 import filteringStream from '../util/filteringStream'
 
 // add levelInString to include DEBUG | ERROR | INFO | WARN
+// outputMode options: short|long|simple|json|bunyan
 const formatOut = filteringStream(bformat({outputMode: 'bunyan', levelInString: true}));
```

Comment:
I was thinking, maybe we should check to see if `NODE_ENV` is dev and make it human readable?  Either that, or use a different stream for stdout that makes it human readable in console, but uses the bunyan format (or json) to output to file.

Not really relevant to your PR, just figure it would be a nice quality of life thing.

Reply:
Totally agree. I might open up a separate PR from this one to update that. I'll remove the comment from this as well.

Reply:
:+1: